enhancements/cli-in-cluster-management.md

```diff
@@ -0,0 +1,166 @@
+---
+title: cli-in-cluster-management
+authors:
+  - "@sallyom"
+reviewers:
+  - TBD
+  - "@soltysh"
+approvers:
+  - TBD
+  - "@soltysh"
+creation-date: 2019-12-03
+last-updated: 2020-03-24-20
```

Comment:
Dang, which universe that date is from :wink:

Reply:
:laughing: quarantine 2020